### ISSUE TITLE
Add public Foundation events calendar

### ISSUE THREAD

**Issue by ElenaFdR** [publiccodenet/http#39]
Containing: 

* our own public events
* events we're attending 
* meetings we're having that are not open to the public but give an overview of who we're working with and what we're working on (for example, "meeting on codebase governence on Decidim")

MVP: Google calendar embedded in the marketing site (HTTP repo). We could use the existing 'community events' calendar for this.

This would be maintained by us, and updated weekly by the comms coordinator.

This is based on @Ainali and my [calendar user needs and design session](https://hackmd.io/tlr3IGFSQsOXRx9ZXFpWfA), which also spawned [About issue #527 for a yearly team schedule](https://github.com/publiccodenet/about/issues/527).

**Comment by bvhme** [comment]
We now have one on `about`, would it be meaningful to link that?

**Comment by ElenaFdR** [comment]
If we host this on Discourse, we could add a 4th category of community-suggested events (events that our community suggests because they're interesting to other community members).

**Comment by lscheske** [comment]
I think it would be great to have something like this. 

Especially regarding events we are attending, it would be great to specify who is attending, and what the goal of attending is. Having a goal of why we are attending things can help prioritize attendance, in case it ever gets really busy.

**Comment by Ainali** [comment]
> Especially regarding events we are attending, it would be great to specify who is attending, and what the goal of attending is. Having a goal of why we are attending things can help prioritize attendance, in case it ever gets really busy.

That is part of the [Attending events](https://about.publiccode.net/activities/events/attending-events.html) already. What's missing is mostly a combined overview.

**Comment by lscheske** [comment]
> That is part of the [Attending events](https://about.publiccode.net/activities/events/attending-events.html) already. What's missing is mostly a combined overview.

Yes, my point is to have it in the overview (e.g. it could even be a excel-type sheet that we all share). The "Attending Events" is more of a guidance for staff on how/when to attend events.

**Comment by ElenaFdR** [comment]
Tangentially related, potentially interesting for seeing how the calendar is populated and shared: [a shared Google calendar of virtual civic tech events](https://twitter.com/codeforamerica/status/1276213987017814016).
I've signed up and will report back.

**Comment by Ainali** [comment]
> Tangentially related, potentially interesting for seeing how the calendar is populated and shared: [a shared Google calendar of virtual civic tech events](https://twitter.com/codeforamerica/status/1276213987017814016).

I wouldn't want us to require people to submit their email to see the calendar, I'd rather we publish our calendar publicly for everyone to see.
